ci-operator/step-registry/cucushift/installer/rehearse/baremetalds/ipi/ovn/lvms/cucushift-installer-rehearse-baremetalds-ipi-ovn-lvms-workflow.yaml

```diff
@@ -0,0 +1,17 @@
+workflow:
+  as: cucushift-installer-rehearse-baremetalds-ipi-ovn-lvms
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-baremetalds-ipi-provision
+    - chain: storage-conf-csi-optional-topolvm
+    post:
+    - chain: cucushift-installer-rehearse-baremetalds-ipi-deprovision
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
+        NETWORK_TYPE=OVNKubernetes
+        VM_EXTRADISKS=true
+        VM_EXTRADISKS_LIST="vda vdb"
```

Comment:
Just trying to understand, here we are not specifically setting `LVM_CLUSTER_DEVICE_PATH`?

Reply:
Yeah, correct.